charts/super-agent-deployment/templates/_helpers.tpl

```diff
@@ -2,15 +2,15 @@
 Return the name of the configMap holding the Super Agent's config. Defaults to release's fill name suffiexed with "-config"
 */ -}}
 {{- define "newrelic-super-agent.config.name" -}}
-{{- .Values.config.name | default (include "newrelic.common.naming.truncateToDNSWithSuffix" ( dict "name" (include "newrelic.common.naming.fullname" .) "suffix" "config" )) -}}
+{{- .Values.config.superAgent.name | default (include "newrelic.common.naming.truncateToDNSWithSuffix" ( dict "name" (include "newrelic.common.naming.fullname" .) "suffix" "agent-config" )) -}}
```

Comment:
agent-> superagent? otherwise don't we confuse it with the other ones?

Reply:
Done on [#c7436bf](https://github.com/newrelic/helm-charts/pull/1221/commits/c7436bf5473024ce7a3034236bc97169695ae943)